cli/kaos_cli/commands/build.py

```diff
@@ -122,6 +136,60 @@ def build(backend: BackendFacade, cloud, env, force, verbose, yes, local_backend
         handle_exception(e)
 
 
+# build list
+# =============
+@build.command(name='list', short_help='List currently deployed backend builds')
+@pass_obj(BackendFacade)
+def list_all(backend: BackendFacade):
+    """
+    List all deployed kaos backend infrastructures.
+    """
+    try:
+        available_contexts = backend.list()
+        if len(available_contexts) > 0:
```

Comment:
Two questions:
  * What if the `available_contexts` is not a list?
    + This will throw a `TypeError: object of type 'NoneType' has no len()`
    + Either handle that case here or make sure that the `backend.list()` function returns a guaranteed list object, either empty (which means no contexts) or containing whatever contexts available
  * What if the `available_contexts` is empty? What would you return back to the user?

Reply:
Adding the 'no valid context found' warning message was missed out. This has been added now. Thanks